src/main/java/org/opensearch/neuralsearch/processor/normalization/ScoreNormalizationTechnique.java

```diff
@@ -20,4 +21,5 @@ public interface ScoreNormalizationTechnique {
      */
     void normalize(final NormalizeScoresDTO normalizeScoresDTO);
 
+    void validateCombinationTechnique(final ScoreCombinationTechnique combinationTechnique) throws IllegalArgumentException;
```

Comment:
nit: in future we can create a dto class for validation context, single instance of CombinationTechnique is this scope